include/vkgcDefs.h

```diff
@@ -398,15 +398,31 @@ struct ResourceMappingData {
 /// Represents graphics IP version info. See https://llvm.org/docs/AMDGPUUsage.html#processors for more
 /// details.
 struct GfxIpVersion {
-  unsigned major;    ///< Major version
-  unsigned minor;    ///< Minor version
-  unsigned stepping; ///< Stepping info
+  static const unsigned Unspecified = ~0U;
 
-  // GFX IP checkers
+  unsigned major = 0;              ///< Major version
+  unsigned minor = Unspecified;    ///< Minor version
+  unsigned stepping = Unspecified; ///< Stepping info
+
+  /// GFX IP checkers
   bool operator==(const GfxIpVersion &rhs) const {
+    if (rhs.minor == Unspecified && rhs.stepping == Unspecified)
+      return major == rhs.major;
+
+    if (rhs.stepping == Unspecified)
+      return std::tie(major, minor) == std::tie(rhs.major, rhs.minor);
```

Comment:
What happens if `[lhs.]minor/stepping` is `Unspecified`?

Reply:
Yes, `operator==` should be symmetric. In general, it's probably clearer to just write this as
`(major == rhs.major) && (minor == rhs.minor || minor == Unspecified || rhs.minor == Unspecified) && ...`